src/components/Auth/Login.js

```diff
@@ -1,6 +1,53 @@
+import axios from "axios";
 import React from "react";
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuthContex } from "../../contex/auth-contex";
+
 import "./auth.css";
 const Login = () => {
+  const navigateHome = useNavigate();
+  const { authDispatch } = useAuthContex();
+  const [userDetail, setUserDetail] = useState({
+    email: "",
+    password: "",
+  });
+  const testUser = {
+    email: "adarshbalak@gmail.com",
+    password: "adarshBalaki123",
+  };
+  const setUserDetailHandler = (e) => {
+    let name = e.target.name;
+    let value = e.target.value;
```

Comment:
Also liked that you used same function call for both of the input fields. Good job.